src/main/java/org/opensearch/security/OpenSearchSecurityPlugin.java

```diff
@@ -2065,6 +2099,17 @@ public void onNodeStarted(DiscoveryNode localNode) {
         if (!SSLConfig.isSslOnlyMode() && !client && !disabled && !useClusterStateToInitSecurityConfig(settings)) {
             cr.initOnNodeStart();
         }
+
+        // create resource sharing index if absent
+        rmr.createResourceSharingIndexIfAbsent();
+
+        for (ResourcePlugin resourcePlugin : OpenSearchSecurityPlugin.GuiceHolder.getResourceService().listResourcePlugins()) {
+            String resourceIndex = resourcePlugin.getResourceIndex();
+
+            this.indicesToListen.add(resourceIndex);
+            log.info("Preparing to listen to index: {} of plugin: {}", resourceIndex, resourcePlugin);
```

Comment:
Same, debug?

Reply:
This one can remain as is to allow operators to understand what resource plugin indices are being listened upon.